orderable/tests/models.py

```diff
@@ -1,5 +1,15 @@
+from django.db import models
+
 from ..models import Orderable
 
 
 class Task(Orderable):
     """A basic orderable model for tests."""
+
+
+class SubTask(Orderable):
+    """An orderable model with unique_together."""
+    task = models.ForeignKey('Task')
+
+    class Meta:
+        unique_together = ('task', 'sort_order')
```

Comment:
Likely - we need a formset validation to catch it. If it is needed, I think we should be able to make that method automatic though so you don't have the manually add it every time. I'd like to work out what is going on and *why* that is needed first though!